proposals/lip-0052.md

```diff
@@ -2272,14 +2272,13 @@ Let `genesisBlockAssetBytes` be the `data` bytes included in the block assets fo
         }
     )
     ```
-    maintaining the lexicographical ordering of the user substore by `NFTEntry.owner` and, for each `NFTEntry.owner`, by `NFTEntry.nftID`.
   
   - If `NFTEntry.owner` has length `LENGTH_CHAIN_ID` bytes, create an entry in the escrow substore with: 
 
     ```python
     storeKey = NFTEntry.owner + NFTEntry.nftID
+    storeValue = `EMPTY_BYTES`
```

Comment:
@shuse2 Just to confirm, an empty object serialized with the empty schema corresponds to EMPTY_BYTES?

Reply:
yes, encoding with empty schema will result in `Buffer.from([])`

Reply:
Than all good, can be resolved